backend/main.py

```
@@ -1,21 +1,10 @@
-    allow_methods=["*"],  # Allow all methods (GET, POST, etc.)
-    allow_headers=["*"],  # Allow all headers
-)
-
-# ✅ Create tables
-Base.metadata.create_all(bind=engine)
-
-# ✅ Include routers
-app.include_router(auth.router, prefix="/auth", tags=["Authentication"])
\ No newline at end of file
+@app.get("/")
+def root():
```

Ruff (0.8.2)

Missing return type annotation for public function `root`

Add return type annotation: `str`

(ANN201)